docs/content/reference/upgrade_dependencies.html.markdown

```diff
@@ -0,0 +1,18 @@
+# Upgrading Dependencies
+
+## Procedure
+
+1. Please have an instance of `magic-modules` available. It will be used later in this procedure.
```

Comment:
[Avoid please](https://developers.google.com/style/tone#politeness-and-use-of-please) in documentation. Additionally, setup like this should be in a [Before you begin](https://cloud.google.com/guides/docs/task#document_the_prerequisites) section.

```suggestion
## Before you begin

1. [Set up your development environment]({{< ref "/get-started/generate-providers/" >}})
```

Reply:
done